services/grafana/client.go

```diff
@@ -352,6 +353,26 @@ func (c *Client) findAnnotations(ctx context.Context, from, to time.Time) ([]ann
 	return response, nil
 }
 
+type grafanaHealthResponse struct {
+	Commit   string `json:"commit"`
+	Database string `json:"database"`
+	Version  string `json:"version"`
+}
+
+// Check calls Grafana API to check its status
+func (c Client) Check(ctx context.Context) error {
+	var status grafanaHealthResponse
+	if err := c.do(ctx, "GET", "/api/health", http.Header{}, nil, &status); err != nil {
+		return fmt.Errorf("error calling Grafana API: %s", err)
+	}
+
+	if strings.ToLower(status.Database) != "ok" {
+		logrus.Errorf("grafana is up but the database is not ok. Database status is %s", status.Database)
+	}
```

Comment:
See https://github.com/percona/pmm-managed/pull/360#discussion_r395740362

We should not return details, by we still should return the fact that Grafana is not ok.

Probably it is easier to return bool.